### PR TITLE
Fix #402

### LINKED ISSUE #402: Improve doco of system properties
Re: https://github.com/OpenHFT/Chronicle-Core/#system-properties-from-file

This doco is out of date - there is an alternative to calling Jvm.init() in a static block - we can use Jvm.getInteger etc. as added by @alamar for #354

It would be also useful to refer to the javadoc for Chronicle's opinionated boolean system properties - `net.openhft.chronicle.core.Jvm#getBoolean(java.lang.String)`

### PR DESCRIPTION
**Pull request by j4sm1ne96**
@alamar I have added you in too as you added in the code below and hence may have advice regarding this readMe: 

https://github.com/OpenHFT/Chronicle-Core/blob/ea/src/main/java/net/openhft/chronicle/core/ChronicleInit.java

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -40,19 +40,8 @@ look in `..` too.
 This is a useful feature if you want to de-clutter your command line.
 
 Chronicle Core class `Jvm` loads the file `system.properties` into the System's properties.
-To ensure it is loaded early enough, add the following code to your `Main`
```
Comment (alamar) [inline README.adoc:43]: Why did you remove this section? I think it's still relevant. And the next one too. I suggest asking @peter-lawrey

**Review by rogersimmons — APPROVED** [review]

File: README.adoc [hunk 2]
```diff
@@ -39,7 +39,16 @@ if it can find this file in the current directory. If it can't find this file in
 look in `..` too.
 This is a useful feature if you want to de-clutter your command line.
 
-Chronicle Core class `Jvm` loads the file `system.properties` into the System's properties.
+Chronicle Core class `Jvm` loads the file `system.properties` into the System's properties. To ensure it is loaded early enough, add the following code to your Main
```
Comment (JerryShea) [inline README.adoc]: "add the following code to your Main" -> "you can load in a static block"

Also please mention how Jvm.getInteger etc work and how they ensure that system.properties are loaded
